tools/Dockerfile

```diff
@@ -105,7 +105,7 @@ RUN copy-item C:\tools\python\python.exe C:\tools\python\python2.exe
 # Install pywin32 for python 2
 #--------------------------------------------------------------------
 
-ENV PYWIN32_VERSION 228
+ENV PYWIN32_VERSION 300
```

Comment:
It seems like this isn't working for python 2.7, maybe they haven't updated the version for 2.7 past 228?

Reply:
Yea I looked at pip and it has the version for 300 being Python 3.x only. Update just that one or keep it all at 228 for consistency?

Reply:
Probably just update the one, at this point we should probably consider python 2 and python 3 as being totally separate.